build/.moduleignore

```diff
@@ -145,11 +145,9 @@ xterm-addon-*/fixtures/**
 xterm-addon-*/out/**
 xterm-addon-*/out-test/**
 
-@improbable-eng/**
```

Comment:
Should not we keep these?

Reply:
it's still there in `.webignore`, server part doesn't use this package

Reply:
oh good, sorry